src/text.h

```diff
@@ -387,6 +395,19 @@ struct Text {
                 --s.cursorend;
             };
     }
+    void DeleteWord(Selection &s) {
+        SelectWord(s);
+        if (!RangeSelRemove(s))
+            if (s.cursor < (int)t.Len()) { t.Remove(s.cursor, 1); };

```

Comment:
Call `Delete`? Or call `SelectWord` in the caller, and remove this function entirely?